docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.mdx

```diff
@@ -53,35 +58,90 @@ The icon should be somewhat in line with the contemporary icon styles (i.e. not
 
 The name should ideally be no longer than 15 characters, and must be shorter than 20 characters.
 
+:::danger Bad examples
+
+- Test app for testing stuff
+- A test app (Special Edition)
+  :::
+
 ### Just the name
 
-The name should be just the name, without any additional info. For example, don't append what the app does to the name (e.g. `Apostrophe - Markdown Editor`), and use the summary to provide this extra information instead.
+The name should be just the name, without any additional info. For example, don't append what the app does to the name, and use the summary to provide this extra information instead.
+
+:::danger Bad examples
+
+- Apostrophe - Markdown Editor
+- Apostrophe (Markdown Editor)
+- Apostrophe (Special Edition)
+  :::
 
 ### No weird formatting
 
 The name should not have any weird formatting or punctuation. For example, it should not be all-lowercase, all-uppercase, camel case, or contain dashes or periods. Cases where the weird formatting is part of the brand are exempt from this (e.g. `VLC`).
 
+:::danger Bad examples
+
+- apostrophe
+- APOSTROPHE
+- AposTrophe
+- Apos.trophe
```

Comment:
I'd maybe replace the last two with more realistic examples, e.g. "ApostropheEditor" and "Apostrophe.org"